### PR TITLE
Climber subsystem

### LINKED ISSUE #62: Basic climb subsystem

### PR DESCRIPTION
**Pull request by ThatXliner**
- https://www.notion.so/warriorborgs/Climb-81612f33f2bc4c828dd8fca9294b868e
 - Closes #62 

I still don't have

 - [ ] Logging (AdvantageKit), probably out of scope though
 - [x] Make the code simulation-ready
 - [x] Add constants boilerplate for each TelescopicArm
 - [x] Constants boilerplate for the Climber
 - [x] Implement the commands
 - [x] Add subsystem to RobotContainer

### CODE REVIEW

**Review by realjoshuau — COMMENTED** [review]
vid in sim / irl & will approve

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -27,62 +32,30 @@ public final class Constants {
   public static final double kWheelBase = Units.inchesToMeters(20.75);
   public static final double kWheelDiameter = 0.1016;
   public static final double kWheelCircumference = kWheelDiameter * Math.PI;
+  public static final RobotMode kRobotMode = RobotMode.SIM;
```
[the remark this answers is not in the document]
Comment (ThatXliner) [reply src/main/java/frc/robot/Constants.java:35]: Merge conflict stuff prob

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -86,6 +96,28 @@ public RobotContainer() {
...
+    }
+    climb =
+        new Climb(
+            new TelescopicArm(Constants.ClimbConstants.leftTelescopicArmConstants),
+            new TelescopicArm(Constants.ClimbConstants.rightTelescopicArmConstants),
+            gyro);
+    operator.povDown().onTrue(new StrangleClimbers(climb));
+    operator.povUp().onTrue(new ReleaseClimbers(climb));
+    // XXX: Not sure if this is a good idea
+    //        climb.setDefaultCommand(new StrangleClimbers(climb));
```
[the remark this answers is not in the document]
Comment (ThatXliner) [reply src/main/java/frc/robot/RobotContainer.java:118]: Yea fab L

**Comment by ThatXliner** [comment]
I will probably close and re-open a take-2 PR due to merge conflicts

**Pull request closed without merging**